vit-servicing-station-tests/src/common/startup/server.rs

```diff
@@ -39,7 +39,23 @@ impl ServerBootstrapper {
         self
     }
 
-    pub fn start(&self) -> Result<Server, ServerBootstrapperError> {
+    pub fn start(&self, token: &str) -> Result<Server, ServerBootstrapperError> {
+        let server = self.start_async()?;
+        self.is_up(&server, token)?;
+        Ok(server)
+    }
+
+    pub fn is_up(&self, server: &Server, token: &str) -> Result<(), ServerBootstrapperError> {
+        for _ in 0..3 {
+            if server.is_up(token.to_string()) {
+                return Ok(());
+            }
+            std::thread::sleep(std::time::Duration::from_secs(1));
+        }
+        Err(ServerBootstrapperError::FailToBootstrap)
+    }
```

Comment:
I think `start` should just start the server and should know nothing about the token. `is_up` can be called explicitly with the token in the test.

A following call would look like:
```rust
ServerBootstrapper::new()
...
.start().unwrap()
.is_up(token)
```